### ISSUE TITLE
Feature suggestions - QoS toggles, device tracker options 

### ISSUE THREAD

**Issue by Tuxrug** [AlexandrErohin/home-assistant-tplink-router#74]
While the device tracker for LAN devices from the router is very useful, not all users might have a need for it and prefer to not log all devices' status, so I suggest an option in configuration to skip creation of these entities if unneeded, or only scan for new LAN devices to track on demand. 

If possible, however, would it be possible to add a global QoS toggle? Even when no devices are scheduled to be prioritized, my AX3000 seems to reserve a decent chunk of WAN bandwidth as long as QoS is enabled globally so I would love to be able to automate QoS as a whole when I don't need anything prioritized. If it's possible to add a prioritize toggle on devices (if creating entities for LAN devices as above is enabled in setup), that could be useful too.

**Comment by JoseCaetanoNeto** [comment]
It would be very useful to have the option to turn off device tracking. Excellent suggestion, I vote for it.

**Comment by jkabradley** [comment]
I agree, this would be a great option. Right now every wifi IoT device (like light switches) shows up as a device tracker.  They can be disabled, but I'd prefer to not create them in the first place.

**Comment by lucianf** [comment]
Is this feature planned at all? Would be great to have the option to disable creation of device trackers
